lib/dolly/callbacks.rb

```diff
@@ -0,0 +1,20 @@
+module Dolly
+  module Callbacks
+    extend ActiveSupport::Concern
```

Comment:
avoid dependencies with rails, please remove active support concerns and use the pure ruby method